lib/git.js

```diff
@@ -52,6 +52,10 @@ exports.getCommits = function (options) {
     commit.category = parsed[3] || '';
     commit.subject = parsed[4];
 
+    if (raw.indexOf('BREAKING CHANGE') !== -1) {
```

Comment:
Instead of needing to add the exact text `BREAKING CHANGE`, I think it would be better if you can just have `breaking change` within the commit message. So we should downcase `raw` before checking `indexOf`.

Also can you pull the `BREAKING_CHANGE_INDICATOR` (i.e. `breaking change`) as a constant up at the top.